pkg/repro/repro.go

```diff
@@ -638,7 +638,7 @@ func (ctx *context) testImpl(inst *vm.Instance, command string, duration time.Du
 		return false, fmt.Errorf("failed to run command in VM: %v", err)
 	}
 	rep := inst.MonitorExecution(outc, errc, ctx.reporter,
-		vm.ExitTimeout|vm.ExitNormal|vm.ExitError)
+		vm.ExitTimeout|vm.ExitNormal)
```

Comment:
But if a C repro exits with non-zero status, it won't be a kernel bug.
I am not sure if syz-execprog exits with non-zero status, but if it does, it's also not a kernel bug.
There is something I don't understand here. Perhaps we mishandle vm.ExitError in vm.MonitorExecution? Or confuse it with "lost connection". Then the question is if it's possible to distinguish error exit status and "load connection"? Or maybe vm.ExitError always means "lost connection"? But then in what context we want to ignore it?...

Reply:
Hm, if I'm reading the code correctly, AFAIU `vm.ExitError` happens when one of the merger inputs fails with an error other than `EOF`, but I'm not sure when exactly that can happen. Perhaps when `ssh` loses connection?..

Reply:
I am not sure what happens with stdout pipe when a command fails... If the pipe read also fails then it will fail when both (1) connection is terminated and (2) program itself exits with non-0 status.